internal/controllers/inventory_controller_test.go

```diff
@@ -33,7 +33,7 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/reconcile"
 
-	oranv1alpha1 "github.com/openshift-kni/oran-o2ims/api/v1alpha1"
+	oranv1alpha1inv "github.com/openshift-kni/oran-o2ims/api/inventory/v1alpha1"
```

Comment:
Missed a rename here

Reply:
I will run another end to end install today to validate the renaming. Thank you!